rpm-ostree-container.ks.in

```diff
@@ -0,0 +1,32 @@
+#test name: rpm-ostree
+# https://github.com/rhinstaller/anaconda/pull/4617
+
+# Use the default settings.
+%ksappend common/common_no_payload.ks
+
+# Set up the RPM OSTree source.
+ostreecontainer --no-signature-verification --transport=registry --stateroot=fedora-silverblue --remote=fedora-silverblue --url=quay.io/fedora/fedora-silverblue:rawhide
+
+%post
+# Check the name of the remote.
+name="$(ostree remote list)"
+
+#if [ "${name}" != "fedora-silverblue" ]; then
+if ! ostree remote list | grep -q "fedora-silverblue"; then
+    echo "Unexpected name fedora-silverblue in \"${name}\"" >> /root/RESULT
+fi
```

Comment:
- Maybe move the `name=$(...` part into the if?
- You also want to drop the commented out bit :)

Reply:
inlined - thanks for spotting this.